Makefile

```diff
@@ -46,15 +47,20 @@ ifneq ($(buildx), True)
 define build
 	docker build -t apache/$(1)-$(3):$(version) \
 		--build-arg checkout_v=$(checkout) \
+		--build-arg version=$(version) \
+		--build-arg debug=$(debug) \
 		--build-arg IMAGE_BASE=$(image_base) \
 		--build-arg IMAGE_TAG=$(image_tag) \
 		--build-arg CODE_PATH=$(4) \
+		--build-arg DEBUG=$(debug) \
```

Comment:
@EnableAsync two arguments for opt-in the `debug` variable.

Reply:
Thanks for your kind reminder, it has been updated.